src/test/scala/com/ibm/couchdb/api/DocumentsSpec.scala

```diff
@@ -263,6 +267,12 @@ class DocumentsSpec extends CouchDbSpecification {
         documents.getMany.includeDocs[FixXPerson].byTypeSimple(FixViews.typeFilter, fixDesign.name,
```

Comment:
I wonder if  there really much benefit in having a byType[K, V] in the first place. The type filter view serves the sole purpose, type based filtering, and has specific requirements to work well. I wonder if it makes more sense to be prescriptive about the structure and content of such a view to ensure it works as expected. 

The argument against I suppose might be that a user might want to reuse a view designed for something else, but that sounds like it might present more problems down the line